test/powershell/Modules/Microsoft.PowerShell.Management/Test-Connection.Tests.ps1

```diff
@@ -7,7 +7,7 @@ Describe "Test-Connection" -tags "CI" {
     BeforeAll {
         $oldInformationPreference = $InformationPreference
         $oldProgressPreference = $ProgressPreference
-        $InformationPreference = "Ignore"
+        $InformationPreference = "SilentlyContinue"
```

Comment:
Why does `Ignore` need to be updated to `SilentlyContinue` for this test?

Reply:
The breaking change for `ActionPreference.Ignore` is very temporary, and a result of juggling a few related PRs while trying to keep their changes separate from one another (I've had such changes grouped in other PRs before and received pushback to separate them, so this is a result of that).

PR #10238 (which is waiting for review, hint, hint) removes the restriction around `ActionPreference.Ignore`. The removal of that restriction was approved in [this discussion in issue 4348](https://github.com/PowerShell/PowerShell/issues/4348#issuecomment-490672315).

Once we get both this PR and #10238 reviewed/merged, we'll be good.

Reply:
Thanks for pointing me to #4348. It makes sense to remove the check for `ActionPreference.Ignore`.
But I think you probably should incorporate the fix of `Remove unnecessary restriction around ActionPreference.Ignore` into this PR.

The PR #10238 is a feature change that seems not approved by the committee yet (I looked at  #6702 and your RFC), so it might take some time for it to get through.

Reply:
I thought you might say that.

Related to PR #10238, see [this note from @SteveL-MSFT]([https://github.com/PowerShell/PowerShell/issues/6702#issuecomment-515147136]). That's not the committee, that's just Steve, but he did say "**we** would welcome a PR from the community", which certainly sounds like a welcome feature change to me.

Also I appreciate the suggestion that #10238 might take some time to get through. Ultimately I'm all for getting things done quickly, so if this PR must be held back because of the change being in that other PR, I'll move it over.

I should call out though that you collectively, as a team, request quite a bit of PR content juggling -- sometimes suggesting PRs should be combined, other times suggesting PRs should be split up, and others suggesting that content is moved from one PR to another. I try to make logical decisions with where PR changes go, ensuring that I only group like changes with other like changes, but the suggestions still come up. I'm really not pointing fingers here, and I'm not mad. I just feel I should call out that it's really not encouraging, and I feel it gets in the way of getting things done.

Reply:
I cannot speak for the overall guidelines/rules for contribution. I know @joeyaiello is actively working on that, and I'm sure it will be clearer and more transparent hopefully soon.
As a PR reviewer, I spot issues in the PR, and in this case, a breaking change. A breaking change will massively slow down the accepting of a PR because it will have to go through the committee again.
What I suggested here was a shortcut to get the PR merged faster, since once the minor fix is moved here, it will be an already-approved change given committee's review in https://github.com/PowerShell/PowerShell/issues/4348#issuecomment-490672315.

Reply:
I agree the changes for `ActionPreference.Ignore` are better here. I wanted to avoid updating a few PRs, and was hoping to see both of them move forward, but since that's not likely I'll go for the route with the least amount of friction. I've brought the changes removing the restriction on `ActionPreference.Ignore` over here, making this non-breaking. I've also brought over the corresponding tests from the other PR, reverted the change to the `Test-Connection` test in this PR that was working around the breaking change, and will update the title and OP of this PR with the extra details in a moment.

You can remove the breaking change label now, @daxian-dbw.

Reply:
As per the other comment, I brought over the change to get this merged in faster.